zest/releaser/tests/test_setup.py

```diff
@@ -58,9 +62,16 @@
     # clearly.  This catches Fore.RED and Fore.MAGENTA.
     # Note the extra backslash in front of the left bracket, otherwise
     # you get: "error: unexpected end of regular expression"
-    (re.compile(r'\x1b\[31m'), 'RED '),
-    (re.compile(r'\x1b\[35m'), 'MAGENTA '),
-])
+    (re.compile(re.escape(Fore.RED)), 'RED '),
```

Comment:
Ah, I wanted to do something like this when I added color support, but did not know how. Thanks!